cores/esp8266/Arduino.h

```diff
@@ -224,12 +224,13 @@ void loop(void);
 void yield(void);
 void optimistic_yield(uint32_t interval_us);
 
-#define digitalPinToPort(pin)       (0)
-#define digitalPinToBitMask(pin)    (1UL << (pin))
+#define _PORT_GPIO16    1
+#define digitalPinToPort(pin)       ((pin)==16)?(_PORT_GPIO16):(0)
```

Comment:
@earlephilhower request to surround the entire ternary is not done. This should be as follows:
` (((pin)==16)?(_PORT_GPIO16):(0))`

Same is valid for all new defines below.